csharp/ql/src/semmle/code/csharp/controlflow/internal/NonReturning.qll

```diff
@@ -55,10 +55,23 @@ private class ThrowingCall extends NonReturningCall {
   override ThrowCompletion getACompletion() { result = c }
 }
 
+/** Holds if accessor `a` has an auto-implementation. */
+private predicate hasAccessorAutoImplementation(Accessor a) {
```

Comment:
Why does the more natural `not exists(a.getBody())` not work?

Reply:
I think it will now, given the changes in this PR.